lgc/patch/PatchCopyShader.cpp

```diff
@@ -299,19 +299,16 @@ void PatchCopyShader::collectGsGenericOutputInfo(Function *gsEntryPoint) {
         Value *output = callInst->getOperand(callInst->getNumArgOperands() - 1); // Last argument
         auto outputTy = output->getType();
 
-        unsigned value = cast<ConstantInt>(callInst->getOperand(0))->getZExtValue();
+        InOutLocationInfo outLocInfo = {};
+        outLocInfo.location = cast<ConstantInt>(callInst->getOperand(0))->getZExtValue();
         const unsigned streamId = cast<ConstantInt>(callInst->getOperand(2))->getZExtValue();
-
-        GsOutLocInfo outLocInfo = {};
-        outLocInfo.location = value;
-        outLocInfo.isBuiltIn = false;
         outLocInfo.streamId = streamId;
 
-        auto locMapIt = resUsage->inOutUsage.outputLocMap.find(outLocInfo.u32All);
-        if (locMapIt == resUsage->inOutUsage.outputLocMap.end())
+        auto locInfoMapIt = resUsage->inOutUsage.outputLocInfoMap.find(outLocInfo.u16All);
+        if (locInfoMapIt == resUsage->inOutUsage.outputLocInfoMap.end())
           continue;
 
-        unsigned location = locMapIt->second;
+        unsigned location = reinterpret_cast<InOutLocationInfo *>(&locInfoMapIt->second)->location;
```

Comment:
The same about using a u16all applies here as well.

Reply:
Got it.